File: C# Net/VIQA/HtmlElements/Interfaces/ICheckList.cs
using System.Collections.Generic;

namespace VIQA.HtmlElements.Interfaces
{
    public interface ICheckList : ISelector<Checkbox>
    {
        void CheckGroup(params string[] values);
        void UncheckGroup(params string[] values);
        void CheckOnly(params string[] values);
        void UncheckOnly(params string[] values);
        List<string> GetListOfChecked();
        List<string> SelectedItems { get; }
    }
}

File: Java/src/main/java/VIElements/Interfaces/ICheckList.java
package VIElements.Interfaces;

import java.util.List;

/**
 * Created by roman.i on 29.09.2014.
 */
public interface ICheckList {
    void checkGroup(String... values) throws Exception;
    void uncheckGroup(String... values) throws Exception;
    void checkOnly(String... values) throws Exception;
    void uncheckOnly(String... values) throws Exception;
    void clear(String... values) throws Exception;
}
